rcv/frame_handlers/frc_2019.py

```diff
@@ -74,10 +73,9 @@ def prep_output_image(self):
             output = cv2.circle(output, self._points[1], 3, (0, 255, 0), -1)
         self._output_frame = output
 
-    def show_images(self):
+    def get_sliders(self):
```

Comment:
This is a bad name for a function

Reply:
I agree but i didn't know how to call it